best-practices/README.md

```diff
@@ -110,6 +110,10 @@ Testing
 
 * Avoid `any_instance` in rspec-mocks and mocha. Prefer [dependency injection].
 * Avoid `its`, `let`, `let!`, `specify`, and `before` in RSpec.
+  * Instead of using these, extract a method that you can pass the thing that
```

Comment:
These are the only sub-bullets in the guides. Is there a way we can flatten this?